### DISCUSSION TITLE
magick.net questions

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#636] [Help]
Hello, I'm using MAGICK.NET There are some questions I would like to ask you. I wonder if it is convenient for you?
Sorry, my English is not very good
 
I want to use MAGICK.NET How to replace the region color of the image?
I'm using draw now, but I don't think it's very efficient

**Comment by dlemstra** [comment]
Can you clarify what the region color is? And feel free to share some of your current code so I can understand beter what you are trying to do.

**Reply by jueqing1015** [reply]
For example, I specify a rectangular area on the image and fill it with white

**Reply by jueqing1015** [reply]
Drawables s = new Drawables();
              s.Rectangle(new Rectangle(0, 0, 100, image.Height));
              s.FillColor(Color.White);
                  s.Draw(image);
I'm using this method now

**Comment by dlemstra** [comment]
This is actually a pretty efficient method to do this. Another method would be to create an image that is `100 x image.height` and the use `Composite` to draw it over the other image. Not sure which one is faster though.

**Reply by jueqing1015** [reply]
Thank you very much!
